cdn.py

```diff
@@ -25,6 +27,25 @@ def fastly_purge(*args):
             run("curl -s -X PURGE {0}{1} | grep 'ok'".format(hostname, govuk_path.strip()))
 
 
+@task
+@runs_once
+def purge_all_document_attachments(arg):
+    "Purge each of the attachments for a specific document."
+    if env.environment == 'production':
+        hostname = 'https://www.gov.uk/api/content/'
+    elif env.environment == 'staging':
+        hostname = 'https://www-origin.staging.publishing.service.gov.uk/api/content/'
```

Comment:
This is an origin URL rather than a Fastly one. Is this right or should it be the same as line 17?